test/util/ethereum.ts

```diff
@@ -211,4 +209,19 @@ export const createTransaction = async (
     from: alith.address,
     privateKey: ALITH_PRIVATE_KEY,
   };
-  
\ No newline at end of file
+  
+/// Same as tx.signAndSend(account), except that it waits for the transaction to be included in a block:
+/// 
+/// ```
+/// const txHash = await tx.signAndSend(alice);
+/// // We don't know if the transaction has been included in a block or not
+/// const { txHash, blockHash } = await signAndSendAndInclude(tx, alice);
+/// // We know the blockHash of the block that includes this transaction
+/// ```
+export async function waitUntilIncluded(promise, web3, txHash)  {
+  while ((await customWeb3Request(web3, "eth_getTransactionByHash", [
+    txHash.result,
+  ])).result.blockNumber== null) {  
+    await promise;
```

Comment:
In Rust this would only wait the first time, not sure how it works in javascript